README.md

```diff
@@ -140,13 +140,8 @@ We test all combinations of:
 
 ## Known issues and Limitations
 
-- The [SendCtrlAltDel](docs/api_requests/actions.md#sendctrlaltdel) API request
-  is not supported for aarch64 enabled microVMs.
-- If a CPU template is not used on x86_64, overwrites of `MSR_IA32_TSX_CTRL` MSR
-  value will not be preserved after restoring from a snapshot.
 - The `pl031` RTC device on aarch64 does not support interrupts, so guest
```

Comment:
Is there a way to remove/move the entire section?

Reply:
yes, I was aiming to do that, but couldn't find a place for the ARM/RTC issue. Do you have a suggestion in mind?

Reply:
Maybe an arm.md or clocks.md? But they would be pretty short at the moment :)

Reply:
yeah, doesn't seem to be worth it.

Reply:
We can have a Limitations.md for generic stuff that doesn't fit anywhere else but then the argument would be why there and not here so, I don't have a good suggestion right now. I'm ok with the current change.